app/api/v1/redflags/models.py

```diff
@@ -0,0 +1,97 @@
+import datetime
+from flask import request
+from flask_restful import reqparse
+
+"""This module handles incidence data model"""
+
+db = [] #incidence
+parser = reqparse.RequestParser(bundle_errors=True)
+
+parser.add_argument('createdBy',
+                    type=str,
+                    required=True,
+                    help="Please fill this field"
+                    )
+
+parser.add_argument('location',
+                    type=str,
+                    required=True,
+                    help="Please fill this field"
+                    )
+
+parser.add_argument('images',
+                    action='append',
+                    help="Please fill this field"
+                    )
+
+parser.add_argument('videos',
+                    action='append',
+                    help="Please fill this field"
+                    )
+
+parser.add_argument('comment',
+                    type=str,
+                    required=True,
+                    help="Please fill in this field"
+                    )
+
+class Incidence:
+    def __init__(self, createdBy, location,image, video, comment, incidence_type): 
+        """creates instance variables for the Incidence class"""
+        self.counter = 0
+        self.status = 'draft'
+        self.createdOn = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+        self.image = image
+        self.video = video
+        self.createdBy = createdBy
+        self.location = location
+        self.comment = comment
+        self.incidence_type = incidence_type
+
+    def get_all_incidences(self):
+        return db
+    
+    def create_incidence(self):
+        args = parser.parse_args()
+        data = {}
+        data['id'] = len(db)+1
+        data['status'] = self.status
+        data['createdOn'] = self.createdOn
+        data['image'] = self.image
+        data['video'] = self.video
+        data['createdBy'] = self.createdBy
+        data['location'] = self.location
+        data['comment'] = self.comment
+        data['incidence_type'] = self.incidence_type
+
+        if data["id"] == "keyerror":
+            return "keyerror"
+        
+        self.db.append(data)
+        return self.id
+
+        return {'message':'successfully created',
```

Comment:
This code is unreachable. Because of the above return self.id, also after creating a new object the status code should be `201`